android/src/toga_android/window.py

```diff
@@ -147,17 +147,18 @@ def get_visible(self):
     ######################################################################
 
     def get_window_state(self, in_progress_state=False):
-        decor_view = self.app.native.getWindow().getDecorView()
-        system_ui_flags = decor_view.getSystemUiVisibility()
-        if system_ui_flags & (
-            decor_view.SYSTEM_UI_FLAG_FULLSCREEN
-            | decor_view.SYSTEM_UI_FLAG_HIDE_NAVIGATION
-            | decor_view.SYSTEM_UI_FLAG_IMMERSIVE
-        ):
-            if self._in_presentation_mode:
-                return WindowState.PRESENTATION
-            else:
-                return WindowState.FULLSCREEN
+        if getattr(self, "app", None) is not None:
```

Comment:
Since, get_window_state() is being called in window_cleanup, so it might be getting called after window.close() is called on the window. Hence, leading to a situation where self.app might not exist.